TempForceSensor/CS/README.md

```diff
@@ -0,0 +1,234 @@
+---
+page_type: sample
+urlFragment: temp-force-sensor
+languages:
+  - csharp
+products:
+  - Windows 10
+  - Windows IoT 
+  - Windows 10 IoT Enterprise
+description: Control a basic temperature and force sensor for Windows 10 IoT Enterprise.
+---
+
+# Temperature and force sensor
+This sample uses SPI communication. 
+
+A temperature/force sensor is connected to an ADC, the ADC is connected to the UP Board through SPI Pins. 
+
+The ADC converts the analog sensor output to a digital value that is then read by the UP Board using SPI. 
+
+The value read from the ADC is displayed on the screen attached to the UP Board.
+
+This is basically a simplified version of Potentiometer sensor sample, which has an LED light as an extra output.
+
+You can also use a Force sensor in this sample. Try to press the Force sensor gentle or hard to see the data output difference.
+
+This sample only has C# version.
+
+## Read before start
+This sample assumes that UP Board has already been set up as below:
+
+- UP Board has been connected to HDMI monitor
+- A SD image card has been plugged to UP Board

```

Comment:
Does UP Board even have an SD card slot?  This line should be removed I think.

Reply:
Removed the line